src/lib/containers/table/SynapseTable.tsx

```diff
@@ -995,7 +1009,22 @@ export default class SynapseTable extends React.Component<
         )
       }
     } else {
-      return <p className={isBold}> {columnValue} </p>
+      const [displayString, isShortened] = getShortString(columnValue)
+      if (!isShortened) {
+        return <p className={isBold}> {columnValue}</p>
+      } else {
+        return (
+          <p className={isBold}>
+            <ImageButtonWithTooltip
+              tooltipText={columnValue}
+              callbackFn={_.noop}
```

Comment:
if callbackFn is an optional prop can't it be left out instead of passing _.noop?

Reply:
No because the absence of calllbackFn will make the component render the dropdown since that's how it decides if it's a button or a dropdown trigger. For a button we do expect some kind of callback since most the the time there is some kind of action to perform. If this seems hacky to you I can add another prop to component to explicitly flag that it is not a dropdown trigger. I don't have strong feelings either way

Reply:
Ah I see, I would push towards making ImageButtonWithTooltip have a third option to not render as a button since clicking doesn't have an action for this case, the component could be renamed to something like ElementWithTooltip.

Granted, there are other parts of SRC that aren't perfect in this regard (anchors that don't have hrefs etc.).